### ISSUE TITLE
Investigate error with ArcGIS remote layer

### ISSUE THREAD

**Issue by giohappy** [GeoNode/geonode-mapstore-client#846]
The client throws an error with [this remote ArcGIS dataset](https://master.demo.geonode.org/catalogue/#/dataset/1161).
The information required by the client seems to be correctly returned by the API. It looks like a failure related to the bbox ccords.

![image](https://user-images.githubusercontent.com/571129/155292913-97602bb5-33f5-4b33-8628-38b62d4e18be.png)

**Comment by DavidQuartz** [comment]
@giohappy, after investigating and consulting with @allyoucanmap, I've found that the issue is from how we are handling cases when a layer's extent goes beyond the standard Web Mercator max extent which we are using inside Geonode. One possible fix would be to add additional checks inside [FitBounds](https://github.com/GeoNode/geonode-mapstore-client/blob/master/geonode_mapstore_client/client/js/plugins/FitBounds.jsx#L23-L28) to make sure we always have a consistent extent.

The error Openlayers sends back seems to be rather that no extent was provided, but I found that that was not the case.

Another thing I found was that there may be a wrong conversion inside GeoNode from the ArcGIS bbox to geonode bbox.
